TddShop/TddShop.Cli.Tests/Shipment/AncientRomeShippingServiceTests.cs

```diff
@@ -13,13 +14,184 @@ namespace TddShop.Cli.Tests.Shipment
     public class AncientRomeShippingServiceTests
     {
         private Mock<IDeliveryService> _deliveryService;
+        private Mock<INumeralsConvereter> _numeralsConvereter;
         private AncientRomeShippingService _target;
 
         [SetUp]
         public void Initialize()
         {
             _deliveryService = new Mock<IDeliveryService>();
-            _target = new AncientRomeShippingService(_deliveryService.Object);
+            _numeralsConvereter = new Mock<INumeralsConvereter>();
+            _target = new AncientRomeShippingService(_deliveryService.Object, _numeralsConvereter.Object);           
         }
+        [Test]
+        public void ShipOrder_IfOrderContainsAtLeast1item_DeliveryServiceShouldReturnShipmentReferenceNumberOnce()
```

Comment:
To jest trochę typ testu, który mocno "betonuje" implementacje. Tak naprawdę testujesz stuba.